benchexec/tools/graves-par.py

```diff
@@ -0,0 +1,78 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.coveriteam as coveriteam
+import benchexec.result as result
+from benchexec.tools.template import UnsupportedFeatureException
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
+import re
+
+class Tool(coveriteam.Tool):
+    """
+    Tool infor for Graves, a verifier selector based on Graph Neural Networks 
+    We inherit Coveriteam's infrastructure. 
+    """
+
+    REQUIRED_PATHS = [
+        "coveriteam",
+        "bin",
+        "lib",
+        "depends",
+        "predict",
+        "predict/build",
+    ]
+
+    def name(self):
+        return "Graves-Par"
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("graves.sh")
+
+    def version(self, executable):
+        return self._version_from_tool(executable)
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
```

Comment:
It seems this was copied from `coveriteam.Tool` but as is likely contains a bug. For CoVeriTeam, the executable is in a subdirectory, so here `parent_dir=True` needs to be used, but for Graves-Par the executable is in the main directory, so now the paths apply to the wrong directory.

Reply:
I see. I have changed it